core/modules/layout/layout.module

```diff
@@ -1737,6 +1738,12 @@ function layout_get_handler_name($plugin_type, $plugin_name) {
       else {
         $class_name = 'BlockLegacy';
       }
+      if ($delta == 'field_block' && isset($parts[2])) {
+        $field_name = $parts[2];
+        if (empty(field_get_block_list()[$field_name])) {
```

Comment:
I know, we don't have to be compatible with php 5.3 anymore, but syntax-wise this looks odd (function inside function as array). An intermediate variable might look cleaner.

Reply:
Thanks. I've added an intermediary variable.